include/picongpu/plugins/transitionRadiation/TransitionRadiation.hpp

```diff
@@ -219,43 +303,36 @@ namespace picongpu
                  * transition radiation calculation and create a folder for transition
                  * radiation storage.
                  */
-                void pluginLoad() override
+                void init()
```

Comment:
why is this now an `init()` and no longer `pluginLoad()`?

Reply:
It was previously `pluginLoad()` because this is an inherited and implemented function in `ILightweightPlugin`. By switching to `plugins::multi::IInstance`, this function does not exist anymore. I prefer `init` slightly over `pluginLoad` and it's probably clearer, that this is now not function from the parent class